LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2018, MIT Data To AI Lab
+Copyright (c) 2022, DataCebo, Inc.
```

Comment:
Hi, I commented in the sdv slack some questions around the license and some dependencies. Is that the right channel or should I raise an Issue?

Reply:
@eedeleon this PR is on a different topic. We are aware of your message on slack.